jaqpotpy/__init__.py

```diff
@@ -1,3 +1,4 @@
 from .jaqpot import Jaqpot
 
 
+__version__ = '1.2.0'
```

Comment:
The current version of jaqpotpy that we use is 6.1.0. You can check it on the releases section of the github repo:
![image](https://github.com/ntua-unit-of-control-and-informatics/jaqpotpy/assets/4619900/0dabcef6-fea9-4c0a-a796-3417eabd8906)

Reply:
Actually, I think that i shouldn't commit this change. It is only necessary for installing and running jaqpotpy from my local path. So this change is not supposed to be uploaded on public repo.